docs/apache-airflow/administration-and-deployment/production-deployment.rst

```diff
@@ -43,14 +43,9 @@ Once that is done, you can run -
 
 .. code-block:: bash
 
-    airflow db upgrade
+    airflow db migrate
 
-``upgrade`` keeps track of migrations already applied, so it's safe to run as often as you need.
-
-.. note::
-
-    Do not use ``airflow db init`` as it can create a lot of default connections, charts, etc. which are not
-    required in production DB.
+``migrate`` keeps track of migrations already applied, so it's safe to run as often as you need.
```

Comment:
It would be great to add in ` .. note` here that formerly it was `upgrade`. (before 2.7)  some people will look at latest version of the docs and try with older airflow version - note here and in other places of the .rst docs might be useful.

Reply:
Added notes in a few important places for context.